<a href="https://colab.research.google.com/github/suryakanthPS/DS_Mini-Projects/blob/main/As3_Q1_Hypo_test_Cutlet.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
#import libraries
import pandas as pd
import numpy as np
import scipy
from scipy import stats

#### Cutlet

*   Ho: μ1 = μ2 (There is no difference in diameters of cutlets between two units).
*   Ha: μ1 ≠ μ2 (There is significant difference in diameters of cutlets between two units) 
* 2 Sample 2 Tail test applicable

In [ ]:
(7.019091429-6.964297143)/(np.sqrt(((0.2884**2)/35)+((0.3434**2)/35)))

0.7228784684535277

In [ ]:
data = pd.read_csv("/content/Cutlets.csv")

In [ ]:
UnitA = pd.Series(data.iloc[:,0])
UnitB = pd.Series(data.iloc[:,1])

In [ ]:
# 2 sample 2 tail test 
#stats.ttest_ind(array1,array2) - calculates t test for two independent sample scores
p_value=stats.ttest_ind(UnitA,UnitB)
p_value

Ttest_indResult(statistic=0.7228688704678063, pvalue=0.4722394724599501)

In [ ]:
p_value[1]

0.4722394724599501

* pvalue is 0.47 which is greater than significance lvl 0.05
* Result - Accept Null hypo UnitA=UnitB
* Inference - There is no significant difference in cutlet size of both units